data/tlg0527/tlg048/greekLit.tlg0527.tlg048.opp-eng2.xml

```diff
@@ -0,0 +1,5887 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader>
+    
+    <fileDesc>
+<titleStmt><title>Isaias</title>
```

Comment:
Since this title is not English, it should also have an `xml:lang` attribute. Is this Latin?